code/__DEFINES/components.dm

```diff
@@ -433,3 +433,5 @@
 #define COMSIG_XENO_TURF_CLICK_SHIFT "xeno_turf_click_shift"				//from turf ShiftClickOn(): (/mob)
 #define COMSIG_XENO_TURF_CLICK_CTRL "xeno_turf_click_alt"					//from turf AltClickOn(): (/mob)
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"				//from monkey CtrlClickOn(): (/mob)
+
+#define COMSIG_ATOM_ON_BREW "oh, shit, i'm sorry"
```

Comment:
~~sorry for what~~
why do you do this. just do what everything else does.

Reply:
linter failed due to no newline here, just add it and travis should be content